packages/synapse-react-client/src/components/EntityCitation/EntityCitation.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import { Meta, StoryFn, StoryObj } from '@storybook/react'
+import EntityCitation from './EntityCitation'
+import { useTheme, Box } from '@mui/material'
+
+const RenderComponent = (StoryComponent: StoryFn) => {
+  const theme = useTheme()
+  return (
+    <Box sx={{ backgroundColor: theme.palette.primary.main, padding: '20px' }}>
+      <StoryComponent />
+    </Box>
+  )
+}
```

Comment:
Better name for this decorator, and another style for accuracy

```suggestion
const ProjectHeaderStyleDecorator = (StoryComponent: StoryFn) => {
  const theme = useTheme()
  return (
    <Box sx={{ backgroundColor: theme.palette.primary.main, color: 'white', padding: '20px' }}>
      <StoryComponent />
    </Box>
  )
}
```